test/integration/helpers/serverspec/xpack_upgrade_spec.rb

```diff
@@ -18,7 +18,7 @@
 
   describe 'security roles' do
     it 'should list the security roles' do
-      roles = curl_json('http://localhost:9200/_xpack/security/role', username='es_admin', password='changeMeAgain')
+      roles = curl_json("http://localhost:9200/_xpack/security/role", username='es_admin', password='changeMeAgain')
```

Comment:
Curious why these had to be switched to double quotes. The difference between single and double quotes in ruby is that it allows variable interpolation (which is not happening here). My best bet is that you originally were setting `es_api_scheme` here and removed it later.